plans/rpminspect/README.md

```diff
@@ -0,0 +1,111 @@
+# Rpminspect
+
+<!-- SPHINX-START -->
+
+Run [rpminspect] on the current Copr project or Koji build
+
+## Synopsis
+
+```yaml
+plans:
+  import:
+    url: https://github.com/packit/tmt-plans
+    ref: main
+    name: /plans/rpminspect
+```
+
+## Description
+
+This plan simply runs the command
+
+```console
+$ rpminspect [previous_build] [koji_build/copr_build]
+```
+
+where `koji_build` is the build specified by `RPMINSPECT_KOJI_BUILD`. If this option is not provided, it is assumed
+that the plan is running against a `copr_build`, and the build artifacts are retrieved automatically from testing-farm.
+
+`previous_build` is automatically determined from running
+
+```console
+$ koji list-tagged
+```
+
+:::note
+
+Some functionalities like automatically determining `previous_build` are only available for Fedora packages.
+
+:::
+
+## Options
+
+`RPMINSPECT_KOJI_BUILD`
+
+: :::note
+
+Not yet implemented
+
+:::
+
+Run `rpminspect` on the specified Koji build instead of the expected Copr project
+
+`RPMINSPECT_TESTS`
+
+: Run only the specified inspections. This option has precedence over `RPMINSPECT_EXCLUDE`.
+
+See `rpminspect -l` for a list of available tests.
+
+`RPMINSPECT_EXCLUDE` \[Default: `metadata`\]
+
+: Exclude the specified inspections. This option has no effect if `RPMINSPECT_TESTS` is specified.
+
+See `rpminspect -l` for a list of available tests.
+
+`RPMINPSECT_ARCHES`
+
+: Run inspection only on the specified architecture packages
+
+::: note
+
+Keep in mind the architectures available in the testing-farm runner that runs this job.
+
+Also note that `src`, and `noarch` are also considered "architectures" in this context.
+
+:::
+
+## Examples
+
+- Inspect the upstream packit projects
+  ```yaml
+  plans:
+    import:
+      url: https://github.com/packit/tmt-plans
+      ref: main
+      name: /plans/rpminspect
+  ```
+- Filter `disttag` inspection
+  ```yaml
+  plans:
+    import:
+      url: https://github.com/packit/tmt-plans
+      ref: main
+      name: /plans/rpminspect
+  environment:
+    RPMINSPECT_EXCLUDE: disttag
```

Comment:
Testing if this section is true, will report back

Reply:
I've tried to execute this [here](https://artifacts.dev.testing-farm.io/9d83af17-78b7-42f7-8f16-32d3eaac65ab/), but it didn't work.

@psss @lukaszachy, am I wrong in this understanding?

I have run `tmt plans show` and it reports that `environment` field is present after the plan's data has been expanded, however if I look at the files within `testing-farm`, I don't see any mention of that variable being set.

<details><summary>tmt plans show</summary>

```
/plans/rpminspect
                 summary Run rpminspect on all srpm and binary rpms
             description Equivalent to bohdi job running
                         https://github.com/fedora-ci/rpminspect-pipeline
                discover 
                     how fmf
          dist-git-merge false
                    link 
         dist-git-source false
           dist-git-init false
dist-git-remove-fmf-root false
                  fmf-id false
                 exclude 
           modified-only false
                  filter tag: rpminspect
                   where 
                    test 
               sync-repo false
                 prepare 
                    name Get necessary packages
                     how install
                   where 
                    copr 
               directory 
                 exclude 
                 missing fail
                 package rpminspect
                         rpminspect-data-fedora
                         koji
                         clamav-freshclam
                         tree
                 prepare 
                    name Get other files from repo
                     how shell
                   where 
                  script curl --silent --show-error --fail --retry 5 --output "$TMT_PLAN_DATA/viewer.html" https://raw.githubusercontent.com/rpminspect/rpminspect/main/contrib/viewer.html

             environment RPMINSPECT_EXCLUDE: disttag
                 enabled true
```

</details>

Reply:
The patch proposed in https://github.com/teemtee/tmt/pull/2452 should fix the problem. Thanks for reporting the issue.